src/components/gravid/GravidReducer.ts

```diff
@@ -5,11 +5,7 @@ import { mapValidationResponse } from './mapValidationResponse';
 import { Omplassering } from './Omplassering';
 import { Tiltak } from './Tiltak';
 
-export const validateTiltak = (
-  tiltak: Tiltak,
-  state: GravidState,
-  nextState: GravidState
-) => {
+export const validateTiltak = (tiltak: Tiltak, state: GravidState, nextState: GravidState) => {
```

Comment:
Trodde prehooken til Kent skulle sørge for at disse kom underhverandre. Er det noe nytt her?

Reply:
Noe prettier greier som skjer her :/